.gitlab-ci.yml

```diff
@@ -5,7 +5,13 @@ include:
     file:
       - templates/artifacts/python-library.yaml
 
+variables:
+  BUILD_PY_VERSION: python3.9
+
 tox:
+  parallel:
+    matrix:
+      - LANGUAGE_VERSION: [python3.9, python3.10, python3.11]
```

Comment:
Based on the gitlab log for >python311, you need to bump `BASE_CONTAINER_VERSION` (L19) to latest ([3.1.0](https://github.com/NCI-GDC/base-containers/tree/3.1.0)), 2.3.1 stopped at py311

Reply:
Yay! That does seem to have done the trick. Was gonna leave it for another day but that was an easy enough fix. Thank you!